### ISSUE TITLE
[Re-opening 176] Mekanism Breaks IC2

### ISSUE THREAD

**Issue by aaronfranke** [mekanism/Mekanism#231]
Starting this issue to re-open issue 176, found here: https://github.com/aidancbrady/Mekanism/issues/176

On the FTB forums, Aidan stated this (http://forum.feed-the-beast.com/threads/ftb-central-mod-permissions.3801/#post-43077):

"Mekanism as a whole, although working fine independently, has smooth integration for popular mods such as IC2, BuildCraft, Railcraft and Universal Electricity."

Clearly, Mekanism does NOT have smooth integration with IC2, infact the two mods together are unplayable due to this uncontrollable power drain.

I am aware you guys claim this to be an IC2 bug, but the people on the IC2 forums claim this to be a Mekanism bug. Keeping in mind that there are many bugs from UE mods (Experience with Voltz, Ampz, and making custom modpacks with UE mods), I consider it more likely that it is a Mekanism issue. Also, a user named "Player" on the IC2 forums commented on issue 455 on why this could possibly happen:

"Mekanism is so broken lol...

Mekanism's CableUtils.emitEnergyFromAllSides() doesn't handle the case where no universal cables are nearby (networks is empty) properly.
The return value (totalRemaining) remains 0, which later on sets the amount field in the event to 0, telling the mfsu that the energy packet has been consumed completely.
Due to the global nature of the event, there doesn't have to be anything Mekanism-related nearby.

Besides that the Mekanism code doesn't account for the inherent inaccuracy of floating point operations, causing rounding errors/energy loss in some cases.

Btw. you can easily see if an energy flow is using the IC2 energy network by measuring the MFSU block with the EU Reader."

It would be great if you would work together with the developers of IC2 to fix this bug, nomatter which mod contains this issue, it is not right to claim smooth integration where such a thing is not present. If you could pinpoint a change to the IC2 code to fix this, then we know it is an IC2 issue, if not, it may be a Mekanism issue.

TL;DR, Don't mark it as working when it is not, and please try to fix this very serious bug. I think the user "Player"'s comment may help you.
